test/unit/core.ts

```diff
@@ -837,6 +838,44 @@ describe("snabbdom", function () {
           assert.equal(elm.children[1].innerHTML, "symbol");
           assert.equal(elm.children[2].innerHTML, "symbol");
         });
+        it("simultaneous addition in beginning and removal in end", function () {
```

Comment:
This test also passed before this PR. I'm adding it for symmetry with the test below and to make sure that the non-inefficent case reported in #1099 is now tested.